user/views.py

```diff
@@ -0,0 +1,14 @@
+from django.shortcuts import render
+from django.views.generic.edit import CreateView
+from .forms import Accountform
+from django.views.generic import TemplateView
+
+class Start(TemplateView):
+  template_name = 'start.html'
+
+class Signup(CreateView):
+    form_class = Accountform
+    template_name = "User_HTML/formpage.html" 
```

Comment:
また、`formpage.html`というよりは、signUpをするtemplateになると思うので、
```suggestion
    template_name = "user/signup.html" 
```
に変更するのはいかがでしょうか？


すでにsignup.htmlがありますが、おそらくそのhtmlはユーザ登録が完了した先の画面だと思いますので、`successed_signup.html`など登録が完了したことがわかる名前にするといいと思います！